API/controllers/controller.js

```diff
@@ -0,0 +1,16 @@
+const { fetchTable } = require("../models/model");
+
+const ENV = process.env.NODE_ENV || "development";
+
+const dataPath = `${__dirname}/../../db/data/${ENV}-data`;
```

Comment:
Why are you directing to ENV files here? This should be in your connection file.